specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

```diff
@@ -738,6 +738,46 @@
         }
       }
     },
+    "/operations/": {
```

Comment:
I understand that this is the path the API currently uses for polling URLs but I wonder if we should make it more specific (e.g. `/operations/snapshot/create` or something) to be future-proof it in case other long-running operations are added.

Reply:
This is the URL right now.
`https://endpoint.io/operations?api-version=2022-11-01-preview\u0026snapshot=snapshot-1678937759509`

Service has to change if we want to be more specific here, right?

Reply:
Yes, this is feedback to the service team to consider. /cc @JeffreyRichter

Reply:
@jimmyca15

Reply:
LRO status is generic resource. In such case our thought is that the design of the endpoint should be generic as well.

Reply:
If we're being pedantic about REST conventions, the proposed `/operations/snapshot/create` doesn't follow the "collection/resource" pattern.  It would need to be `operations/snapshot` and "create" would be implied by the verb used.  The alternative would be something like `operations/create-snapshot`.

Reply:
FWIW, I didn't see a one-size-fits-all endpoint for status monitoring like this before so I wanted to make sure that future-proofing is taken into consideration. Even something like `/snapshots/operations` feels much safer to me. That being said, I defer to the architects on this @tg-msft @johanste @JeffreyRichter

Reply:
Our data-plane guidance is here: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs. The URL for the operation COLLECTION is not LISTable and so there is no polymorphism issue. Customers can only GET a single operation and they know its type. 

I'm not sure I'm addressing your question but I think I am.